modules/relay/src/main/RelaySync.scala

```diff
@@ -1,6 +1,7 @@
 package lila.relay
 
 import cats.syntax.traverse.*
+import cats.syntax.foldable.*
```

Comment:
I cannot just use `cats.syntax.all.*` here because it is conflicted with `study.*` import.

Because we export `OptionSyntax` with this: https://github.com/lichess-org/lila/blob/master/modules/common/src/main/Lila.scala#L39